Mathlib/Tactic/MoveAdd.lean

```diff
@@ -0,0 +1,448 @@
+/-
+Copyright (c) 2023 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Arthur Paulino, Damiano Testa
+-/
+import Mathlib.Algebra.Group.Basic
+
+/-!
+
+#  `move_add` a tactic for moving summands in expressions
+
+The tactic `move_add` rearranges summands in expressions.
+
+The tactic takes as input a list of terms, each one optionally preceded by `←`.
+A term preceded by `←` gets moved to the left, while a term without `←` gets moved to the right.
+
```

Comment:
Does this tactic depend on how the operator is associated? That is: does a goal `a + (b + c)` work differently from `(a + b) + c`?

Reply:
Could you add a comment somewhere explaining this?